Doc/library/sys.rst

```diff
@@ -545,15 +545,15 @@ always available.
    | :const:`mant_dig`   | DBL_MANT_DIG   | float precision: the number of base-``radix``    |
    |                     |                | digits in the significand of a float             |
    +---------------------+----------------+--------------------------------------------------+
-   | :const:`max`        | DBL_MAX        | maximum representable positive finite float      |
+   | :const:`!max`       | DBL_MAX        | maximum representable positive finite float      |
```

Comment:
I think this is better done in a separate PR, so we can backport to the other bugfix branches.

Reply:
Yes, revert the commit here. It doesn't need an issue; trivial docs fixes don't need an issue. You can just attach the second PR to the same issue.

Reply:
Also, based on @zware's comment on the bug, it's going to be better to change these to just put `max` in double backticks, since they aren't supposed to link anywhere.

Reply:
That comment was @merwok, not me :wink:.  I agree with him, though.